packages/ui-components/src/components/tag-letter/usage/angular.md

```diff
@@ -0,0 +1,10 @@
+```html
+<!-- Default -->
+<kv-tag-letter label="Tag Letter" />
+
+<!-- With Color -->
+<kv-tag-letter label="Tag Letter" color="green" />
+
+<!-- With Tag Letter -->
+<kv-tag-letter label="Tag Letter" tag-letter="T" />
```

Comment:
in angular you can actually refer to the tagLetter property instead of the tag-letter attribute ;)